### DISCUSSION TITLE
Body Materials Not Working

### DISCUSSION THREAD

**Discussion by TsuyuLily** [xivdev/Penumbra#167] [Q&A]
I've tried a few times to figure out why I can't get this to work when I implement mods into Penumbra... I was hoping someone would be able to help me. I know it has something to do with the new Bibo update and I'm stumped. 
![ffxiv_03092022_101614_949](https://user-images.githubusercontent.com/101277675/157484119-6128d4d4-d736-4fe5-b8f4-86d2580b049c.png)

**Comment by Kylie-Digital** [comment]
Check [this ](https://github.com/xivdev/Penumbra/discussions/164#discussioncomment-2304098)too for further help.

b -> bibo  for the new Bibo 2.0+
b _(or bibo)_ -> d for converting bibo mods to G3
If your body base is B+, You'll need to use the Compatibility option added in B+ 2.0~

It's important to have your body of choice installed in penumbra since it adds the directories needed for the materials. I and my wife had issues when the body was installed in TexTools.
